### DISCUSSION TITLE
Is stream having a timeout setting

### DISCUSSION THREAD

**Discussion by giiggssxue** [NginxProxyManager/nginx-proxy-manager#2618] [Q&A]
We got a finding that using streaming but after nearly 60 seconds the connection was reset.

Wondering if there is a default timeout for stream as well and if possible to do change.

i am using the docker version of NPM

Many Thanks
Ryan

**Comment by gregfr** [comment]
We are having the same problem. It's probably possible to edit the nginx config manually, but it may be overwritten at some point...
Does anyone know a reliable way to change/add the parameters?

**Comment by gregfr** [comment]
old topic: https://github.com/NginxProxyManager/nginx-proxy-manager/issues/206
